### PR TITLE
Fixed missing reference to the /lib folder in the .iml file. #52

### PR DESCRIPTION
**Pull request by AndreiiiH**
Fixes the missing reference to the /lib folder in the .iml file (issue #52 ). Rebuilt the artifact with the new reference.

### CODE REVIEW

**Comment by rdegges** [comment]
Thank you! <3

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,8 @@ lib/aws-java-sdk-1.10.74.jar
 
 lib/aws-java-sdk-1.11.37.jar
 
+lib/aws-java-sdk-1.11.228.jar
```
Comment (mraible) [inline .gitignore]: Should this be `lib/*` to prevent having to add a new ignore for every release?

Comment (rdegges) [reply .gitignore]: PR? <3

Comment (AndreiiiH) [reply .gitignore]: @mraible I agree.
Also, I have a question in regards to why the iml file is getting ignored?
The fix itself was adding a dependency on RUNTIME to the iml file, then rebuilding the artifact.
I'm just wondering whether this is why the wrong artifact got built.
@rdegges

Comment (rdegges) [reply .gitignore]: That might be the cause. I'm not a Java developer. I've mentioned this in other threads, but I'm maintaining this library by triaging PRs while Okta tries to find a new full time maintainer.

**Pull request merged**